src/components/BookForm.js

```diff
@@ -65,35 +46,28 @@ const BookForm = () => {
         author: newBook.author,
         category: newBook.category,
       })).then(() => {
-        setShowSuccess(true);
+        showToast('success', 'Add New Book', 'New Book Added');
         dispatch(fetchBooks());
       });
       resetForm(e);
     }
   };
 
   return (
-    <div className="w-full mt-5 book-form">
-      <h1 className="text-lg">ADD NEW BOOK</h1>
-      <form onSubmit={submitNewBook} className="flex items-center justify-between gap-x-5">
-        <div className="form-group w-[55%]">
+    <div className="w-full p-4 mt-5 book-form">
+      <Toast ref={toastRef} />
+      <h1 className="text-lg font-semibold text-gray-400">ADD NEW BOOK</h1>
+      <form onSubmit={submitNewBook} className="flex flex-col items-center justify-between md:flex-row gap-x-5 gap-y-5">
+        <div className="form-group w-full lg:w-[55%]">
           <input type="text" className="form-control" id="title" name="title" placeholder="Book title" onChange={(e) => handleChange(e)} required />
         </div>
 
-        <div className="form-group w-[25%]">
+        <div className="form-group w-full lg:w-[25%]">
           <input type="text" className="form-control" id="author" name="author" placeholder="Author" onChange={(e) => handleChange(e)} required />
         </div>
 
-        <button type="submit" className="submit-btn border-2 border-black px-5 py-3 w-[20%]">ADD BOOK</button>
+        <button type="submit" className="submit-btn border-2 bg-primaryBlue border-primaryBlue hover:bg-blue-700 hover:border-blue-700 text-white px-5 py-3 w-full lg:w-[20%]">ADD BOOK</button>
```

Comment:
### Your Design
![image](https://user-images.githubusercontent.com/93039359/230078685-ab69febe-5c0b-4c5d-b8c4-114fa9ecd674.png)

### Zeplin's Design
![image](https://user-images.githubusercontent.com/93039359/230078910-f5fdabf2-331b-41e3-b0c3-5588ee85753b.png)


- [ ] It would be great if your button have a border radius to match the design given 👍 Kindly follow the design to make your project look attractive 👍